core/case_study/views.py

```diff
@@ -307,3 +308,44 @@ def submit_comment(request, case_study_id):
         }
     }
     return JsonResponse(data)
+
+@csrf_exempt
+def submit_report(request, comment_id):
+    comment = get_object_or_404(Comment, pk=comment_id)
+    reasons = request.POST.get('report_reason', None)
+    report_author_name = request.user.get_full_name()
+    report_author_email = request.user.email
+    #create report
+    report = CommentReport.objects.create(comment=comment, 
+      comment_author=comment.user, report_author=request.user,
+      comment_body = comment.comment, comment_date = comment.comment_date, 
+      report_date=timezone.now(), reason=reasons, report_reviewed=False, 
+      report_author_email=report_author_email, report_author_name=report_author_name )
+
+    data = {
+
+      'comment': {
+        'author': report.comment_author.get_full_name(),
+        'body': report.comment_body,
+        'date': report.comment_date
+      },
+
+      'report': {
+        'author': report.report_author.get_full_name(),
+        'date': report.report_date,
+        'reason': report.reason,
+        'report_reviewed': False
+      }
+
+    }
+    return JsonResponse(data)
+
+@csrf_exempt
```

Comment:
Please look up what csrf is before you disable it
https://docs.djangoproject.com/en/2.2/ref/csrf/

This should be @login_required

You can fix this by using `fetch` and adding this to your headers `"X-CSRFToken": getCookie("csrftoken"),`
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API